Cargo.toml

```diff
@@ -34,5 +34,10 @@ std-embedded-nal = "0.1"
 tokio = { version = "1.9", features = ["rt-multi-thread", "time", "macros"] }
 std-embedded-time = "0.1"
 
+[patch.crates-io.std-embedded-nal]
```

Comment:
Nice work patching this as dev-dep and releasing nonetheless! ;)

Reply:
Dev dependencies aren't used by end-users, only locally for testing etc. so I have no issue releasing with dev-dep patches IMO. Very low risk

Reply:
Yes. Just pointing that out.